ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/ImageSingleVulnerabilities.tsx

```diff
@@ -1,13 +1,208 @@
+            isFixable: boolean;
+        }[];
+    };
+};
+
+export const imageVulnerabilitiesQuery = gql`
+    query getImageVulnerabilities($id: ID!) {
+        image(id: $id) {
+            id
+            imageVulnerabilities {
+                severity
+                isFixable
+            }
+        }
+    }
+`;
+
+export type ImageSingleVulnerabilitiesProps = {
+    imageId: string;
+};
+
+function ImageSingleVulnerabilities({ imageId }: ImageSingleVulnerabilitiesProps) {
+    // TODO Needs integration with URL search filter
+    const { data, loading, error } = useQuery<
+        ImageVulnerabilitiesResponse,
+        ImageVulnerabilitiesVariables
+    >(imageVulnerabilitiesQuery, {
+        variables: { id: imageId },
+    });
+
+    const [activeTabKey, setActiveTabKey] = useCveStatusTabParameter();
+
+    const handleTabClick: TabsProps['onSelect'] = (e, tabKey) => {
+        if (isValidCveStatusTab(tabKey)) {
+            setActiveTabKey(tabKey);
+        }
+    };
+
+    let mainContent: ReactNode | null = null;
+
+    if (error) {
+        mainContent = (
+            <Bullseye>
+                <EmptyState variant={EmptyStateVariant.large}>
+                    <EmptyStateIcon
+                        className="pf-u-danger-color-100"
+                        icon={ExclamationCircleIcon}
+                    />
+                    <Title headingLevel="h2">{getAxiosErrorMessage(error)}</Title>
+                    <EmptyStateBody>Adjust your filters and try again</EmptyStateBody>
+                </EmptyState>
+            </Bullseye>
+        );
+    } else if (loading && !data) {
+        mainContent = (
+            <Bullseye>
+                <Spinner isSVG />
+            </Bullseye>
+        );
+    } else if (data) {
+        const vulnerabilities = data.image.imageVulnerabilities;
+        const severityCounts = severityCountsFromImageVulnerabilities(vulnerabilities);
+        const cveStatusCounts = statusCountsFromImageVulnerabilities(vulnerabilities);
+        // TODO Integrate these with page search filters
+        const hiddenSeverities = new Set<VulnerabilitySeverity>(['LOW_VULNERABILITY_SEVERITY']);
+        const hiddenStatuses = new Set<FixableStatus>([]);
+
+        mainContent = (
+            <>
+                <Grid hasGutter>
+                    <GridItem sm={12} md={6} xl2={4}>
+                        <BySeveritySummaryCard
+                            title="CVEs by severity"
+                            severityCounts={severityCounts}
+                            hiddenSeverities={hiddenSeverities}
+                        />
+                    </GridItem>
+                    <GridItem sm={12} md={6} xl2={4}>
+                        <CvesByStatusSummaryCard
+                            cveStatusCounts={cveStatusCounts}
+                            hiddenStatuses={hiddenStatuses}
+                        />
+                    </GridItem>
+                </Grid>
+            </>
```

Comment:
i dont think you need the fragments here if it's already wrapped by a single component (`Grid`)